resources/assets/js/app/instances/paginateableCollectionView.js

```diff
@@ -0,0 +1,57 @@
+var Marionette = require('backbone.marionette');
+
+module.exports = Marionette.CollectionView.extend({
+
+    initialize: function () {
+
+        $(window).on('scroll', this.fetchPage.bind(this));
```

Comment:
this event will be binded again and again on each view creation, you need or unbind it on view destroy or create event mediator with Radio and use Radio to listen scroll events